.prettierrc.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
I think the `check-format` and the `format` commands in package.json depend on the relative path `../../prettierrc.json`.
Is that not affected with the file move?

Same for `prettierignore`.

Reply:
@HarshaNalluru, good catch, will update those as well.

Reply:
[`find template/template/ \( -type d -name .git -prune \) -o -type f -print0 | xargs -0 sed -i 's/\.\.\/\.\.\/\.prettier/..\/..\/..\/.prettier/g'`](https://stackoverflow.com/a/1583282) did the trick.